### ISSUE TITLE
Do not notify maintainers if the maintainers key of a module is not a list of strings

### ISSUE THREAD

**Issue by sbidoul** [OCA/oca-github-bot#126]
Example: https://github.com/OCA/hr/pull/869#issuecomment-673446711

**Comment by Khalid-SerpentCS** [comment]
> Example: [OCA/hr#869 (comment)](https://github.com/OCA/hr/pull/869#issuecomment-673446711)

Solved!

**Comment by bodedra** [comment]
> > Example: [OCA/hr#869 (comment)](https://github.com/OCA/hr/pull/869#issuecomment-673446711)
> 
> Solved!

@Khalid-SerpentCS You have corrected maintainers key for that specific module, not for a bot issue.

This is a bug:

![Screenshot from 2020-08-13 14-21-15](https://user-images.githubusercontent.com/14229503/90188521-72c7e500-dd70-11ea-9740-9fd56c767326.png)

**Comment by luisg123v** [comment]
Is it expected for the bot to notify maintainers when the PR is not green yet?

If no,, this could be solved by a lint check. If yes, it needs to be solved here, but the lint could still be useful.

**Comment by legalsylvain** [comment]
> Is it expected for the bot to notify maintainers when the PR is not green yet?

Well I think so. But could you open a dedicated issue if you think it's necessary ?

**Comment by sbidoul** [comment]
> Is it expected for the bot to notify maintainers when the PR is not green yet?

Yes that's how it was designed. Personally I liked to be notified as soon as the PR is open. It allows me to give early guidance if necessary.
